### PR TITLE
[Chore] Improve release doc

### PR DESCRIPTION
**Pull request by SbloodyS**
## Brief change log

Improve release doc by adding `DSIP` label.

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

## Pull Request Notice
[Pull Request Notice](https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/en/contribute/join/pull-request.md)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

File: tools/release/github/changelog.py [hunk 1]
```diff
@@ -35,6 +35,7 @@ class Changelog:
     key_labels = "labels"
     key_name = "name"
 
+    label_dsip = "dsip"
```
Comment (zhongjiajie) [inline tools/release/github/changelog.py:38]: could you also change priority in docstring in L28, due to we add new section for our release

File: tools/release/release.py [hunk 2]
```diff
@@ -103,4 +103,8 @@ def build_argparse() -> argparse.ArgumentParser:
             "Environment variable `GH_ACCESS_TOKEN` and `GH_REPO_MILESTONE` must provider"
         )
 
-    print(args.func(ENV_ACCESS_TOKEN, ENV_MILESTONE))
+    try:
+        print(args.func(ENV_ACCESS_TOKEN, ENV_MILESTONE))
+    except Exception as e:
+        print(f"Please run 'python release.py -h' to get help, reason: {e}")
+        exit(1)
```
Comment (zhongjiajie) [inline tools/release/release.py]: We could directly check if `func` is in object `args` and reuse `print_help` method from ArgumentParser object to keep our code easier

```py
    if not hasattr(args, 'func'):
        arg_parser.print_help()
        exit(1)
    print(args.func(ENV_ACCESS_TOKEN, ENV_MILESTONE))
```

Comment (SbloodyS) [reply tools/release/release.py]: Fixed. PTAL

File: tools/release/github/changelog.py (hunk 1 above)
Comment (zhongjiajie) [reply tools/release/github/changelog.py:38]: @SbloodyS seems you missed docstring change

Comment (SbloodyS) [reply tools/release/github/changelog.py:38]: Done.

**Review by zhongjiajie — APPROVED** [review]
+1, Thanks 🎉

**Review by Gallardot — APPROVED** [review]

**Review by caishunfeng — APPROVED** [review]

**Pull request merged**